resolvers/projectResolver.ts

```diff
@@ -1027,12 +1027,37 @@ export class ProjectResolver {
       .setParameter('slugs', slugs)
       .getMany()
 
+
     const projectsUpdatedListing = projects.map(project => {
       return { id: project.id, listed: listed }
     })
 
+    const segmentProject = await projects.map( async project => {
+      const user = await User.find({ id: Number(project.admin) })
+
+      if (project.listed === false) {
+
+        analytics.track(
+          'Project unlisted',
+          `givethId-${project.admin}`,
+          {
+            id: project.id,
+            email: user[0].email,
+            title: project.title,
+            LastName: user[0].lastName,
+            FirstName: user[0].firstName,
+            OwnerId: project.admin,
+            slug: project.slug,
+            listed: listed
+          },
+          null
```

Comment:
You can use  `user?.segmentUserId()` instead of null, I saw it used like this in another parts of codes